packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -133,4 +135,11 @@ Will double processing times. Use only for debugging purposes.",
     defaultDescription: String(defaultOptions.chain.safeSlotsToImportOptimistically),
     group: "chain",
   },
+
+  "chain.archiveStateEpochFrequency": {
+    hidden: true,
+    description: "Minimum number of epochs between archived states",
+    type: "number",
+    group: "chain",
+  },
```

Comment:
I think this flag should be public, with maybe a better name. Lighthouse exposes this and its quite useful to fine tune node behaviour

Reply:
Current behavior is annoying in that it isn't "store a state when epoch % N == 0".
It's "maybe store a state after N epochs since the last stored state if the state is finalized".
Not sure what the current Lighthouse behavior is, but I'm guessing its not what we're doing.

Preference to merge this as is (as a hidden option and only semi-supported) since there are people who want this now.
And figure out better behavior later (or do more research to justify the current behavior) and at that point make a public cli option.